src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java

```diff
@@ -1,30 +1,102 @@
 package org.dailyinfo.telegrambot.telegrambot.service;
 
+
+import org.dailyinfo.telegrambot.telegrambot.service.horoscope.*;
+import org.dailyinfo.telegrambot.telegrambot.service.horoscope.Horoscope.*;
 import org.springframework.stereotype.Service;
 import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
 import org.telegram.telegrambots.meta.api.objects.Message;
 import org.telegram.telegrambots.meta.api.objects.Update;
 
 @Service
-public class MessageService {
+public class MessageService extends Horoscope{
+
+        Aquarius aquarius = new Aquarius();
+        Aries aries = new Aries();
+        Cancer cancer = new Cancer();
+        Capricorn capricorn = new Capricorn();
+        Gemini gemini = new Gemini();
+        Leo leo = new Leo();
+        Libra libra = new Libra();
+        Pisces pisces = new Pisces();
+        Saggittarius saggittarius = new Saggittarius();
+        Scorpio scorpio = new Scorpio();
+        Taurus taurus = new Taurus();
+        Virgo virgo  = new Virgo();
```

Comment:
annotation @Autowired  or create private method in this class for thsi strings

Reply:
I fixed it!